expression/column_test.go

```diff
@@ -214,3 +214,26 @@ func (s *testEvaluatorSuite) TestColHybird(c *C) {
 		c.Assert(v, Equals, result.GetString(i))
 	}
 }
+
+func (s *testEvaluatorSuite) TestPadCharToFullLength(c *C) {
+	ctx := mock.NewContext()
+	ctx.GetSessionVars().StmtCtx.PadCharToFullLength = true
+
+	ft := types.NewFieldType(mysql.TypeString)
+	ft.Flen = 10
+	col := &Column{RetType: ft, Index: 0}
+	input := chunk.New([]*types.FieldType{ft}, 1024, 1024)
+	for i := 0; i < 1024; i++ {
+		input.AppendString(0, "xy")
+	}
+	result, err := newBuffer(types.ETString, 1024)
+	c.Assert(err, IsNil)
+	c.Assert(col.VecEvalString(ctx, input, result), IsNil)
+
+	it := chunk.NewIterator4Chunk(input)
+	for row, i := it.Begin(), 0; row != it.End(); row, i = it.Next(), i+1 {
+		v, _, err := col.EvalString(ctx, row)
+		c.Assert(err, IsNil)
+		c.Assert(v, Equals, result.GetString(i))
```

Comment:
Should we add a line like `c.Assert(len(v), Equals, xx)` before this line.

Reply:
ok